cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -296,4 +295,4 @@ def project_Z(self, q, z):
         if np.all(t == u):
             self.omega /= np.sqrt(2)
 
-        self._update_sum(t, u, delta=delta)
+        self.update_sum(t, u, delta=delta)
```

Comment:
why do we need to make `_update_sum` public?

Reply:
Oh I just read your description: you want to expose this so that the gates can call it directly.